dice/apps/rounds/tests/test_views.py

```diff
@@ -55,26 +56,30 @@ def test_roll(self):
             ],
             format='json',
         )
-        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
 
     def test_too_many_roll(self):
-        """Ensure http 403 is returned after third roll."""
-
-        for status_code in [200, 200, 403]:
-            response = self.client_host.patch(
-                f'/api/v1/rounds/{self.game_round.id}/reroll/',
-                data=[
-                    self.game_round.dice1.id, self.game_round.dice2.id
-                ],
-                format='json',
-            )
-            self.game_round.refresh_from_db()
-            self.assertEqual(status_code, response.status_code)
+        """Ensure http 409 is returned after third roll."""
+
+        status_code = (status.HTTP_200_OK, status.HTTP_200_OK, status.HTTP_409_CONFLICT)
+
+        for sc in status_code:
+            with self.subTest():
```

Comment:
Let's add some more context to the `subTest` call, e.g:

```suggestion
        for index, sc in enumerate(status_code):
            with self.subTest(reroll_index=index):
```